pkgs/development/python-modules/poster3/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, pkgs
+, buildPythonPackage
+, fetchPypi
+, python
```

Comment:
```suggestion
```

Reply:
This is needed for accessing python.sitePackages. I see this done in a lot of other packages too. Is there a more preferred way?

Reply:
I have a lot of suggestions, if you use the default build/install phases, then it's not required.